reference/scalability_guide/mongo_cursor_timeout_exception.rst

```diff
@@ -0,0 +1,21 @@
+How to get rid of the MongoCursorTimeoutException
+-------------------------------------------------
+
+In the PIM we always iterate over cursor and paginate them to avoid big queries on the mongo database. But for some low level operation which are not paginable you can encounter the following exception:
```

Comment:
> In the PIM we always iterate over cursor and paginate them to avoid big queries on the mongo database
weird sentence, in Mongo it's always paginated